addon/components/o-s-s/input-container.ts

```diff
@@ -45,7 +47,10 @@ export default class OSSInputContainer extends Component<OSSInputContainerArgs>
   }
 
   get autocomplete(): 'on' | 'off' {
-    return this.args.autocomplete ?? 'on';
+    if (!this.args.autocomplete) {
+      return 'on';
+    }
```

Comment:
Is this really required ? 
Isn't it already handled with the ternary operator below ? 🤔